src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/TrustDataDao.java

```diff
@@ -13,6 +13,9 @@ public class TrustDataDao {
     @Field("name")
     private String trustName;
 
+    @Field("trust_still_involved_in_overseas_entity")
+    private boolean isTrustStillInvolvedInOverseasEntity;
```

Comment:
Just started looking at the SDK change required... I think this might have to be stored as a string, since if it comes in from the web as 'not selected', i.e. null/undefined, we'll need to store it as that. This might be the first case of this, as all the other questions have a response provided by the time the data block is persisted in Mongo. @markpit what do you think?

Reply:
If we have other flags that we send, what do we do in those instances?

Reply:
i would say see what the other pages do but this could be the first one that is optional, would a Boolean class be ok, would a string true or false in the json get converted to a Boolean, and it can be null if no string in json?

Reply:
Yes, I think this is the first case where it's optional - the web client definitely needs to be able to send in a null/undefined/empty string or something to the PUT end-point and get that back too when calling a GET (e.g. when resuming a submission).

A `Boolean` type here might work. @mattch1 can you check this please? You'll need to test with Postman.

Reply:
used cease date: "error": "ceased_date": "1990-03-28"

1) Tested with "trust_still_involved_in_overseas_entity": **"true",**
"error": "trusts.ceased_date must be null",

2) Tested with "trust_still_involved_in_overseas_entity": **true,**
"error": "trusts.ceased_date must be null",

Ceased date left blank

3) Tested with "trust_still_involved_in_overseas_entity": **"false",**
"error": "trusts.ceased_date must not be null",

4) Tested with "trust_still_involved_in_overseas_entity": **false,**
"error": "trusts.ceased_date must not be null",

5) Tested with "trust_still_involved_in_overseas_entity": **null,**
"error": "trusts.trust_still_involved_in_overseas_entity must not be null",

6) Tested with "trust_still_involved_in_overseas_entity": **"null",**
"error": "trusts.trust_still_involved_in_overseas_entity must not be null",

7) Tested with "trust_still_involved_in_overseas_entity": "whatever string",
HTTP 500 errror:  Cannot deserialize value of type `java.lang.Boolean` from String

Reply:
Type now changed to `Boolean` which seems to work, so resolving this comment.